<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Plot</class>
 <widget class="QWidget" name="Tab_Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="push_refresh">
     <property name="text">
      <string>&amp;Refresh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>650</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_trend">
        <property name="title">
         <string>Trend Plot Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>&amp;Y axis label:</string>
           </property>
           <property name="buddy">
            <cstring>combo_yAxis</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_yAxis">
           <item>
            <property name="text">
             <string>Structure</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>&amp;X axis label</string>
           </property>
           <property name="buddy">
            <cstring>combo_xAxis</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combo_xAxis">
           <item>
            <property name="text">
             <string>Structure</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="cb_showDuplicates">
           <property name="text">
            <string>Show &amp;duplicate structures and supercells</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="cb_showIncompletes">
           <property name="text">
            <string>Show &amp;incomplete structures</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cb_labelPoints">
           <property name="text">
            <string>&amp;Label points</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="combo_labelType">
           <item>
            <property name="text">
             <string>Structure #</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_distance">
        <property name="title">
         <string>Structure selection:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Structure:</string>
           </property>
           <property name="buddy">
            <cstring>combo_distHistPC</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_distHistPC"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_plotType">
        <item>
         <property name="text">
          <string>Trends</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Distance Histogram</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plot &amp;Type:</string>
        </property>
        <property name="buddy">
         <cstring>combo_plotType</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Toggle options...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="Avogadro::PlotWidget" name="plot_plot">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::PlotWidget</class>
   <extends>QFrame</extends>
   <header location="global">avogadro/plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>push_refresh</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>983</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
